wrangler-core/src/main/java/io/cdap/directives/row/Flatten.java

```diff
@@ -110,6 +110,11 @@ public void destroy() {
       }
 
       if (max == 0) {
+        for (int i = 0; i < count; ++i) {
```

Comment:
could you please mutate a copy? I don't think it makes a difference here, but it's better not to have side effects.

Reply:
Done

Reply:
done